lang/python/python-cryptography/Makefile

```diff
@@ -8,11 +8,11 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-cryptography
-PKG_VERSION:=2.8
+PKG_VERSION:=2.9
 PKG_RELEASE:=1
 
 PYPI_NAME:=cryptography
-PKG_HASH:=3cda1f0ed8747339bbdf71b9f38ca74c7b592f24f65cdb3ab3765e4b02871651
+PKG_HASH:=0cacd3ef5c604b8e5f59bf2582c076c98a37fe206b31430d0cd08138aff0986e
```

Comment:
just as a note;
not really important

can be done later in a bigger sweep;

if it's not much trouble you can also remove all the conditional deps;
maybe merge with `Package/python-cryptography/Default)`
```
define Package/python3-cryptography
$(call Package/python-cryptography/Default)
  DEPENDS+= \
      +PACKAGE_python3-cryptography:python3 \
      +PACKAGE_python3-cryptography:python3-cffi \
      +PACKAGE_python3-cryptography:python3-six
  VARIANT:=python3
endef
```

Reply:
Yes, I plan to remove the conditional dependencies as well as the `VARIANT:=python3` in one large update later :joy: